src/tests/testHttp1Parser.cc

```diff
@@ -81,6 +81,7 @@ static SBuf Pretty(SBuf raw)
 static void
 testResults(int line, const SBuf &input, Http1::RequestParser &output, struct resultSet &expect)
 {
+    (void)line;
 #ifdef SQUID_DEBUG_TESTS
     std::cerr << "TEST @" << line << ", in=" << Pretty(input) << "\n";
 #endif
```

Comment:
Same as elsewhere: Move to the missing `#else`.

Reply:
done in commit 14dad596b